ViAn/GUI/mainwindow.ui

```diff
@@ -17,35 +17,44 @@
    <double>1.000000000000000</double>
   </property>
   <widget class="QWidget" name="centralWidget">
-   <layout class="QVBoxLayout" name="verticalLayout_2">
+   <layout class="QVBoxLayout" name="verticalLayout">
```

Comment:
I know this is a overall problem in the ui file but these layouts really should be more descriptive. Something along the lines of projectTreeLayout would be better.